### PR TITLE
Feature/test json

### PR DESCRIPTION
**Pull request by agduncan94**
* support for test json/yaml files

- [x] Check that you pass the basic compilation and unit tests by running `grunt`

### CODE REVIEW

**Comment by denis-yuen** [comment]
That's a lot of fail

File: .idea/codeStyleSettings.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (denis-yuen) [inline .idea/codeStyleSettings.xml]: Let's keep IDE specific files (unless hard to create) out of the repo

File: app/docs/getting-started-with-dockstore.md [hunk 2]
```diff
@@ -28,11 +28,11 @@ Now that you have your `Dockerfile` and `Dockstore.cwl` in GitHub, have setup Qu
...
+In the authenticated Web UI, navigate to 'My Tools' to begin managing Docker images imported through your linked account(s). These pages will allow you to quickly register tools that follow a particularly simple format (look below to manual registration for more complex formats). For quick registration, we look through your quay.io images and see if any are setup as [automated builds](https://docs.quay.io/guides/building.html). Using those to track back to your github, bitbucket, or gitlab accounts, we list all pairs of Docker images with git repositories that contain a `Dockstore.cwl` and a `Dockerfile`. When we discover both of these, we create an unregistered entry in the interface below. Note that we also look for `/test.cwl.json` and `/test.wdl.json` for test parameter files respectively, but they are not required for Quick Registration.
```
Comment (denis-yuen) [inline app/docs/getting-started-with-dockstore.md]: Are you using test.cwl.json lower case or upper case? I noticed on the back-end version of this that a number of references to "Test.json" are upper-cased that way. I think the lower case as in this example does make more sense, but let's make the back-end comments consistent

Comment (agduncan94) [inline app/docs/getting-started-with-dockstore.md]: From now on will use the lowercase version.

**Pull request merged**